File: Strings/validAnagrams/validAnagrams.ts
function isAnagramSort(s: string, t: string): boolean {
    if (s.length !== t.length) return false;
    return s.split("").sort().join("") === t.split("").sort().join("");
}

// Example usage
console.log(isAnagramSort("anagram", "nagaram")); // true
console.log(isAnagramSort("rat", "car"));         // false

File: Strings/validAnagrams/validAnagrams.go
package main

import (
	"fmt"
	"sort"
	"strings"
)

func isAnagramSort(s, t string) bool {
	if len(s) != len(t) {
		return false
	}

	sArr := strings.Split(s, "")
	tArr := strings.Split(t, "")

	sort.Strings(sArr)
	sort.Strings(tArr)

	return strings.Join(sArr, "") == strings.Join(tArr, "")
}

func main() {
	fmt.Println(isAnagramSort("anagram", "nagaram")) // true
	fmt.Println(isAnagramSort("rat", "car"))         // false
}
